### DISCUSSION TITLE
Test discussion

### DISCUSSION THREAD

**Discussion by bmorelli25** [elastic/observability-docs#294] [General] [answered]
Hi @elastic/obs-docs.

This is a discussion.

**Comment by bmorelli25** [comment]
Hi Brandon. Thanks for reaching out!

**Reply by dedemorton** [reply]
Coolest thing I've seen today!

**Reply by bmorelli25** [reply]
Oooo, threaded replies!

**Comment by bmorelli25** [comment, marked as answer]
This is the correct answer.

**Reply by dedemorton** [reply]
I like the idea of being able to lock a conversation after an answer is reached. It's common for people to pile on old threads. Sometimes this is useful, but usually it's not.

**Reply by dedemorton** [reply]
And pinning discussions would save us a ton of time on issues that have a lot of back and forth.

**Reply by EamonnTP** [reply]
Loving this discussion!

**Reply by bmorelli25** [reply]
Hey, look who it is!

**Reply by dedemorton** [reply]
This is Brandon's "cool tool of the day" offering.

**Reply by bmorelli25** [reply]
™️

**Reply by EamonnTP** [reply]
He is a machine!

**Comment by ollyhowell** [comment]
> Hi @elastic/obs-docs.
> 
> This is a discussion.

This is a testing discussion.
